contrib/snyk/check-for-cve.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+PWD=/Users/user/.../hypershift/contrib/snyk
+
+COMPONENT_TO_CHECK=$1
+IMAGE_TO_CHECK=$2
+PULL_SECRETS_FILE=$3
```

Comment:
I would default this to a path most folks could have and not necessarily expect it as an arg...
```bash
PULL_SECRETS_FILE=${PULL_SECRET:-$HOME/.pull-secret.txt}
```